mllib/src/main/scala/org/apache/spark/ml/feature/package-info.java

```diff
@@ -66,7 +66,7 @@
  *      RowFactory.create(0, "Hi I heard about Spark", 3.0),
  *      RowFactory.create(1, "I wish Java could use case classes", 4.0),
  *      RowFactory.create(2, "Logistic regression models are neat", 4.0)));
- *  DataFrame df = jsql.createDataFrame(rowRDD, schema);
+ *  Dataset<Row> dataset = jsql.createDataFrame(rowRDD, schema);
```

Comment:
One minor thing, we usually use `ds` instead of `dataset` to name a temporary Dataset in example code. But it's OK here.